image_proc/CMakeLists.txt

```diff
@@ -12,11 +19,16 @@ if(cv_bridge_VERSION VERSION_GREATER "1.12.0")
 endif()
 
 # Dynamic reconfigure support
-generate_dynamic_reconfigure_options(cfg/CropDecimate.cfg cfg/Debayer.cfg cfg/Rectify.cfg cfg/Resize.cfg)
+generate_dynamic_reconfigure_options(
+  cfg/CropDecimate.cfg
+  cfg/Debayer.cfg
+  cfg/Rectify.cfg
+  cfg/Resize.cfg)
 
-catkin_package(
-  CATKIN_DEPENDS image_geometry roscpp sensor_msgs
-  DEPENDS OpenCV
+catkin_package(CATKIN_DEPENDS
+  image_geometry
```

Comment:
Although all of this CMake reformatting is probably best left to the maintainer, I would have formatted as such:

```
catkin_package(
  INCLUDE_DIRS
    include
  LIBRARIES
    ${PROJECT_NAME}
  CATKIN_DEPENDS
    image_geometry
    roscpp
    sensor_msgs
)
```

:stuck_out_tongue:

Reply:
Thanks for the suggestion. I have incorporated it ;).
I will push maintainer to follow this style. I am open to suggestion and review from maintainer, if they consider it inappropriate to merge it with this style.